pinot-common/src/main/java/org/apache/pinot/common/datablock/BaseDataBlock.java

```diff
@@ -107,105 +110,27 @@ public BaseDataBlock(int numRows, @Nullable DataSchema dataSchema, String[] stri
     _numRows = numRows;
     _dataSchema = dataSchema;
     _numColumns = dataSchema == null ? 0 : dataSchema.size();
-    _fixDataSize = 0;
     _stringDictionary = stringDictionary;
-    _fixedSizeDataBytes = fixedSizeDataBytes;
-    _fixedSizeData = ByteBuffer.wrap(fixedSizeDataBytes);
-    _variableSizeDataBytes = variableSizeDataBytes;
-    _variableSizeData = ByteBuffer.wrap(variableSizeDataBytes);
+    _fixedSizeData = PinotByteBuffer.wrap(ByteBuffer.wrap(fixedSizeDataBytes));
```

Comment:
Curios why you decided to use this style since `PinotByteBuffer.wrap(byte[])` is also available. I understand that it is effectively the same.

Reply:
Probably I didn't remember that utility constructor. Changed in 2b688680f834eca7be682516394e819f590f8038